src/Core/tests/DeviceTests.Android/TestInstrumentation.cs

```diff
@@ -32,7 +32,7 @@ public override IEnumerable<TestAssemblyInfo> GetTestAssemblies()
 			foreach (var assembly in TestInstrumentation.TestAssemblies.Distinct())
 			{
 				// this is required to exist, but is not used
-				var path = Path.Combine(FileSystem.CacheDirectory, assembly.GetName().Name);
+				var path = Path.Combine(FileSystem.CacheDirectory, assembly.GetName().Name + ".dll");
```

Comment:
Tests can't run if they are not .dll.... Why? They are not even used at all since I am making a dummy file...